bin/ChakraCore/CMakeLists.txt

```diff
@@ -30,6 +30,7 @@ if(CMAKE_SYSTEM_NAME STREQUAL Linux)
   set(LINKER_END_GROUP
     -Wl,--no-whole-archive
     -Wl,--end-group
+    -static-libstdc++
```

Comment:
Yes. Aprox 1Mb change in size and yes, only for static lib.

Maybe we could have `--embed-static` or something like that for `build.sh` but I didn't want to separate embedded libc++ testing from not embedded one. This is the package we will release and better we use it and test it all the time ?